browser-interface/packages/shared/dao/sagas.ts

```diff
@@ -160,6 +175,35 @@ function* selectRealm() {
   return realm
 }
 
+function* onboardingTutorialRealm() {
+  const profile = yield select(getCurrentUserProfile)
+  if (profile) {
+    const needs_tutorial = RESET_TUTORIAL || !profile.tutorialStep
+    const new_tutorial_feature_flag = getFeatureFlagVariantName(store.getState(), 'new_tutorial_variant')
```

Comment:
Style and conventions:
* use camel case
* use yield select inside sagas
```suggestion
    const newTutorialFeatureFlag = yield select(getFeatureFlagVariantName, 'new_tutorial_variant')
```

Reply:
Correcting. Thanks!